python/spark_package/spark_package.py

```diff
@@ -365,16 +367,17 @@ def publish_release(name, user, token, folder, version, out, zip):
         zip = zip_artifact(folder, name, version, out)
     binary_zip = ""
     if zip is not None:
-        with open(zip) as f:
+        with open(zip, 'rb') as f:
```

Comment:
Yes, I published a spark package with this version whose is bisecting-kmeans-0.1.1. It was going well.